library/common/glfw/embedder.cc

```diff
@@ -57,6 +57,13 @@ struct FlutterEmbedderState {
   // Handles raw key interactions from GLFW.
   // TODO: Revisit ownership model once Issue #102 is resolved.
   std::unique_ptr<flutter_desktop_embedding::KeyEventHandler> key_event_handler;
+
+  // Window framebuffer dimensions, in pixels.
+  int width_px, height_px = 1;
```

Comment:
Why are these statefull instead of passed as parameters? It looks like they are only ever stored just before calling a function and read inside that function.

Reply:
Check, I have inlined SyncWindowMetrics.